hack/app_sre_pr_check.sh

```diff
@@ -4,6 +4,13 @@
 
 set -exv
 
+curl -s https://raw.githubusercontent.com/maorfr/utilities/master/validate_yaml.py > validate_yaml.py
```

Comment:
My thought was - this will probably be duplicated across operators, so i didn't create the script here. Feel free to put it anywhere you find reasonable and close this PR.

Reply:
Following the same standard we have today, can we put this script under the `hack/` folder in the same repo (`openshift/aws-account-operator`)?

Reply:
Once we centralize CI assets for all operators we can move it to another place.

Reply:
done!